src/ethereum_spec_tools/evm_tools/daemon.py

```diff
@@ -59,6 +65,12 @@ def do_POST(self) -> None:
 
 class _UnixSocketHttpServer(socketserver.UnixStreamServer):
     last_response: Optional[float] = None
+    shutdown_timeout: int
+
+    def __init__(self, *args: Any, shutdown_timeout: int, **kwargs: Any) -> None:
+        self.shutdown_timeout = shutdown_timeout
+        self.last_response = time.monotonic()
```

Comment:
Won't this make the first check timeout if there is too much of a delay between creating the instance and starting the thread?